frontend/src/reducers/account.js

```diff
@@ -0,0 +1,49 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+export const account = createSlice({
+  name: 'account',
+  initialState: { 
+    id: null,
+    username: null,
+    email: null,
+    accessToken: null,
+    errors: null, 
+    signup: false
+  },
```

Comment:
You need to store your data in the local storage to avoid logging out your user after refreshing the page.